lib/adapters/idb/idb.js

```diff
@@ -46,14 +47,10 @@ function IdbPouch(opts, callback) {
 
 function init(api, opts, callback) {
 
-  var name = opts.name;
+  var dbName = opts.name;
 
-  var instanceId = null;
-  var idStored = false;
   var idb = null;
-  api._docCount = -1;
-  api._blobSupport = null;
-  api._name = name;
+  api._meta = null;
```

Comment:
We were stocking so much data in this cache, it was starting to look pretty ugly. So I just folded everything into a single `_meta` variable.